CHANGELOG.md

```diff
@@ -77,3 +81,8 @@ be found.
 ### Swarm Mode
 
 + Added support for swarm service isolation mode [moby/moby#34424](https://github.com/moby/moby/pull/34424)
+
+### Packaging
+
++ Added Packaging for Fedora 27 [docker/docker-ce-packaging#59](https://github.com/docker/docker-ce-packaging/pull/59)
```

Comment:
Shouldn't it be **Add** packaging? ... present tense.

Reply:
This was already merged, but can make a PR to correct this!